bears/yaml/TravisLintBear.py

```diff
@@ -24,6 +26,30 @@ class TravisLintBear:
     LICENSE = 'AGPL-3.0'
     CAN_DETECT = {'Formatting', 'Syntax'}
     SEE_MORE = 'https://docs.travis-ci.com/user/travis-lint'
+    CHECK_CONNECTION_URL = 'https://travis-ci.org/'
+
+    @classmethod
+    def check_prerequisites(cls):
+        base_check = super().check_prerequisites()
+        if base_check is not True:  # pragma: no cover
```

Comment:
can't you modify the environment so the check fails? Then you don't need the pragma